pkg/pillar/cmd/zedagent/zedagent.go

```diff
@@ -1805,6 +1840,26 @@ func handleDPCLDelete(ctxArg interface{}, key string, statusArg interface{}) {
 	triggerPublishDevInfo(ctx)
 }
 
+func dpcHasRealChange(dpcList types.DevicePortConfigList) bool {
```

Comment:
Would be good to add a comment saying it clears the fields which we want to ignore in the comparison.

Reply:
ok.